.circleci/config.yml

```diff
@@ -221,28 +92,46 @@ workflows:
       - integration_tests:
           requires:
             - build
-      - build_docker:
-          requires:
-            - integration_tests
+      - hmpps/build_docker:
+          name: build_docker
+          image_name: "quay.io/hmpps/manage-hmpps-auth-accounts"
           filters:
             branches:
-              only: master
-      - deploy_dev:
+              only:
+                - master
+      - hmpps/deploy_env:
+          chart_name: manage-hmpps-auth-accounts
```

Comment:
don't need either of these two as the release / chart / image names are same as project name